server/mailer/Mail.js

```diff
@@ -0,0 +1,200 @@
+class Mail {
```

Comment:
I think we should split this up into multiple classes to make it easy to extend to other types of mailers in the future. Something along the lines of:

```js
class Mail {
  constructor({ to, from }) {
    this.to = to
    this.from = from
  }
  
  sendMail() {
    return this.mailService.sendEmail(this.emailParams).promise()
  }
  
  get mailService() {
    return new AWS.SES({ apiVersion: '2010-12-01' })
  }
  
  get emailParams() {
    return {
      Destination: {
        ToAddresses: this.to,
      },
      Source: this.from,
      Message: {
        Body: {
          Html: {
            Charset: 'UTF-8',
            Data: this.body,
          },
        },
        Subject: {
          Charset: 'UTF-8',
          Data: this.subject,
        },
      },
    }
  }
}
```

Then we can extend the Mail class for our specific needs:

```js
class RegistrationEmail extends Mail {
  constructor(user) {
    super({ to: [process.env.ADMIN_EMAIL], from: process.env.EMAIL_SENDER })
    this.subject = 'New Registered User'
    this.user = user
  }
  
  get body() {
    // some template string
  }
}
```

Then we can call this as:

```js
const registrationEmail = new RegistrationEmail(user)
registrationEmail.sendMail()
```

Reply:
[bd42052](https://github.com/AMP-SCZ/dpdash/pull/457/commits/bd42052b840fbf1079bbd0f09382a5568c3c9a20)